src/electron/platform/android/rule-information-provider.ts

```diff
@@ -14,20 +14,23 @@ export class RuleInformationProvider {
         this.supportedRules = {
             ColorContrast: new RuleInformation(
                 'ColorContrast',
+                'https://accessibilityinsights.io/info-examples/android/color-contrast/',
                 'Text elements must have sufficient contrast against the background.',
                 [link.WCAG_1_4_3],
                 this.getColorContrastUnifiedFormattableResolution,
                 this.includeColorContrastResult,
             ),
             TouchSizeWcag: new RuleInformation(
                 'TouchSizeWcag',
+                'https://accessibilityinsights.io/info-examples/android/touch-size-wcag/',
                 'Touch inputs must have a sufficient target size.',
                 [link.WCAG_1_3_1, link.WCAG_3_3_2],
                 this.getTouchSizeUnifiedFormattableResolution,
                 this.includeAllResults,
             ),
             ActiveViewName: new RuleInformation(
                 'ActiveViewName',
+                'https://accessibilityinsights.io/info-examples/android/touch-size-wcag/',
```

Comment:
should be https://accessibilityinsights.io/info-examples/android/active-view-name